narwhals/_arrow/series.py

```diff
@@ -70,6 +118,45 @@ def __mul__(self, other: Any) -> Self:
         other = validate_column_comparand(other)
         return self._from_native_series(pc.multiply(self._native_series, other))
 
+    def __truediv__(self, other: Any) -> Self:
+        pa = get_pyarrow()
+        pc = get_pyarrow_compute()
+
+        ser = self._native_series
+        native_type = ser.type
+        original_type_is_int = pa.types.is_integer(
+            native_type
+        ) or pa.types.is_unsigned_integer(native_type)
+
+        if original_type_is_int:
+            ser = ser.cast(pa.float64())
```

Comment:
this is what pandas does for pyarrow dtypes

https://github.com/pandas-dev/pandas/blob/262fcfbffcee5c3116e86a951d8b693f90411e68/pandas/core/arrays/arrow/array.py#L108-L122

I'm wondering why they use `safe=False` and also cast the denominator...will check this